src/config.rs

```diff
@@ -8,10 +8,59 @@ use serde_json::Value;
 use std::collections::HashMap;
 use std::mem;
 use std::str::FromStr;
+use std::sync::RwLock;
 use wildmatch::WildMatch;
+use lazy_static::lazy_static;
 
 const SETTINGS_JSON: &str = include_str!("./config/settings.json");
 
+struct GlobalStorage {
```

Comment:
I'm not sure what the correct path is here... I have a global storage, which currently holds only the config store (but other things can be in there as well)... but unfortunately, this doesn't work because it is not safe to use.

Reply:
Yeah, needing `unsafe` feels strange in this case.  I think you can instantiate it some place under `main` and then

1. Copy it once to each thread, assuming it won't change
2. Wrap it in an `Arc` so that it is `Send`